### PR TITLE
[RPGF-43] Project Detail Retro Funding 5

### PR DESCRIPTION
**Pull request by Billy19191**
# Description

[//]: # "Please include a summary of the changes and the related issue. (You can write below without deleting this comment)"
We added a new "Project Detail" page that shows all information for each project in Retro Funding 5.

### What
[//]: # "Briefly describe this feature / fix / refactor (You can write below without deleting this comment)"
In this ticket, we have added many new sections such as
- Overview
- Funding Sources
- GitHub
- Contract Address
- Package
- Link

within the project details page of the current round.

### Why
[//]: # "Describe the reason why we need this feature / fix / refactor or refer to Jira Ticket (You can write below without deleting this comment)"

Currently, our website allows users to discover all projects that sign-up for retro funding 5 but not the details of each project. So, we add news project page details to show the overall information of each project for users to analyze and explore about them.

Jira Ticket: RPGF-43

## Type of change

Please check the relevent option with `x`.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (if needed)
- [x] My changes generate no new warnings
- [x] I have tested on my `Local Machine`

### CODE REVIEW

File: app/(white_navbar)/explore/RetroPGF5/[name]/_component/_Project/OverviewSection.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import { handleCategoryRound5 } from '../../../../../../lib/InitialMount'
+import { iRetroPGF5Project } from '../../../RetroType5'
+export default function OverviewSection({ data }: { data: iRetroPGF5Project }) {
```
[the remark this answers is not in the document]
Comment (Billy19191) [reply app/(white_navbar)/explore/RetroPGF5/[name]/_component/_Project/OverviewSection.tsx]: I just looked into it, there's unused code that hasn't remove from the draft yet.

File: app/(white_navbar)/explore/RetroPGF5/[name]/_component/_Project/OverviewSection.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import { handleCategoryRound5 } from '../../../../../../lib/InitialMount'
+import { iRetroPGF5Project } from '../../../RetroType5'
+
+interface OverviewSectionProps {
```
Comment (wit03) [inline app/(white_navbar)/explore/RetroPGF5/[name]/_component/_Project/OverviewSection.tsx:4]: This should name `iOverviewSection` like interface in other components

File: app/(white_navbar)/explore/RetroPGF5/[name]/_component/_Project/FundingSection.tsx [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+import { iRetroPGF5Project } from '../../../RetroType5'
+import LinkIcon from '@carbon/icons-react/lib/Link'
+import { ConvertStringToTime } from '@/app/lib/utils'
+const fundingSourcesMap = {
+  ventureFunding: 'Venture Funding',
+  grants: 'Grants',
+  revenue: 'Revenue',
+}
+
+function formatGrant(amount: string): string {
```
[the remark this answers is not in the document]
Comment (wit03) [reply app/(white_navbar)/explore/RetroPGF5/[name]/_component/_Project/FundingSection.tsx:6]: `formatGrant` `formatName` and `containsOnlyDigits` can also be put in `/utils` also please delete these functions in Round 4 page and import them from utils

File: app/(white_navbar)/explore/RetroPGF5/[name]/_component/_Project/OverviewSection.tsx (hunk 2 above)
Comment (Billy19191) [reply app/(white_navbar)/explore/RetroPGF5/[name]/_component/_Project/OverviewSection.tsx:4]: That's right!, Thank for flagging this.

**Review by wit03 — APPROVED** [review]
Lgtm go sleep

**Pull request merged**